apps/search-poc/src/components/ResourceDetails/EModel.css

```diff
@@ -0,0 +1,5 @@
+
+.small-table td {
```

Comment:
Maybe put this into the app css because it's a generic feature. 

It might be be better to use `.small-table > td` if it's a direct child.

Reply:
Moved to global styles.
The child is indirect, so no changes have been made in the selector.